snapcraft/plugins/_python/_pip.py

```diff
@@ -98,7 +98,9 @@ def _setup(self):
                 # Using the host's pip, install our own pip and other tools we
                 # need.
                 self.download({'pip', 'setuptools', 'wheel'}, env=env)
-                self.install({'pip', 'setuptools', 'wheel'}, env=env)
+                self.install(
+                    {'pip', 'setuptools', 'wheel', '--ignore-installed'},
```

Comment:
what is the line here `--ignore-installed` is as much a parameter as `install` is (which is a subcommand).
I would like this to be some sort of keyword argument (option list, true/false, or like `build-attributes`)

Reply:
Agreed. I just pushed a commit that makes the API much more explicit via kwargs. Please let me know what you think.